ts/Core/Utilities.ts

```diff
@@ -461,6 +461,41 @@ namespace error {
     export const messages: Array<string> = [];
 }
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Reduces tree-like objects to a simple object with keys in dot syntax.
+ * @private
+ */
+function flat(
```

Comment:
Apparently the `flat` and `unflat` functions are not used at all, so they should be removed. If we need them down the line, only add them to the core if they are actually used by the core.